ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/saas/searchresults/searchresults.html

```diff
@@ -0,0 +1,32 @@
+<sly data-sly-use.model="com.valtech.aem.saas.api.fulltextsearch.SearchResults"></sly>
+<sly data-sly-use.template="/libs/core/wcm/components/commons/v1/templates.html"
+     data-sly-call="${template.placeholder @ isEmpty=wcmmode.edit}"></sly>
+
+<div class="saas-search-result-tab-wrapper hidden"
+     data-sly-test="${!wcmmode.edit}"
+     saas-search-result-tab-wrapper>
+  <div class="saas-filtered-content">
+    <div class="saas-wrap-filtered-content">
+      <div class="search-type search-normal-hit" saas-search-results-list>
```

Comment:
we should change the prefix to `saas-aem-module-` for attributes. classnames as well and we should use BEM